flink/src/main/java/org/apache/sedona/flink/expressions/Functions.java

```diff
@@ -13,6 +13,7 @@
  */
 package org.apache.sedona.flink.expressions;
 
+import org.apache.calcite.interpreter.Scalar;
```

Comment:
Why is the calcite class introduced here?

Reply:
I will remove it, sorry will take care next time.